browser-entry.js

```diff
@@ -104,7 +105,7 @@ Mocha.Runner.immediately = function(callback) {
  */
 mocha.throwError = function(err) {
   uncaughtExceptionHandlers.forEach(function(fn) {
-    fn(err);
+    fn(console.error(options.color('The following errors occurred:' + err)));
```

Comment:
Under what scenario does this show for? Would like to test locally with the example tests.

Reply:
sorry that should have been left alone, restored back to origin code.

Reply:
removed and restored back to original code.